CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This file is used to list changes made in each version of the Java cookbook.
 
 - Upgrade circleci orb to version 2 and add yamllint and markdown lint
 
+## 4.2.0 - 2019-06-20
```

Comment:
Could you update the date here and update the version in the `metadata.rb` to match 4.2?

Reply:
Version in metadata.rb is already 4.2.0 in my repo... should be reflected here.  Are you asking me to just update the date to today's date then with no other changes?

Reply:
Ah shoot - was looking at a specific commit diff that didn't show the metadata bump 👍 yes, just update the date and then I think it looks get - I'll bug @majormoses to get it released

Reply:
Done... looks like it needs an approving review again though.